lib/membrane/rtcp/packet.ex

```diff
@@ -73,15 +74,38 @@ defmodule Membrane.RTCP.Packet do
     with {:ok, %{header: header, body_size: length, padding?: padding?}} <-
            Header.parse(raw_header),
          <<body::binary-size(length), rest::binary>> <- body_and_rest,
-         {:ok, {body, _padding}} <- RTP.Utils.strip_padding(body, padding?),
-         {:ok, packet_module} <- BiMap.fetch(@packet_type_module, header.packet_type),
-         {:ok, packet} <- packet_module.decode(body, header.packet_specific) do
-      do_parse(rest, [packet | acc])
+         {:ok, {body, _padding}} <- RTP.Utils.strip_padding(body, padding?) do
+      case parse_packet(body, header) do
+        {:ok, packet} ->
+          do_parse(rest, [packet | acc])
+
+        {:error, :unknown_packet_type} = error ->
+          Membrane.Logger.warn("""
```

Comment:
This is still very noisy in Chrome. I'm considering switching this to a `debug`.